docker-compose.yml

```diff
@@ -1,69 +1,58 @@
-version: '3.8'
+version: "3.8"
 
 services:
   nginx:
-    build:
-      context: .
-      dockerfile: sys/docker/alpine-nginx/Dockerfile
-      network: host
+    image: nginx:alpine
```

Comment:
this is going to diverge from the actual image we use on prod: `fabiocicerchia/nginx-lua`
it'd be better to use at least the same images, if not the same dockerfile.